### PR TITLE
Switch to bootstrap5

### PR DESCRIPTION
**Pull request by gtema**
- add bootstrap5 content
- switch to grid layout (sidebar left, toctree right, etc)

### CODE REVIEW

**Comment by gtema** [comment]
this PR additionally does a full cleanup of old and unused content

**Comment by tischrei** [comment]
Mmmh, looks a bit shitty; especially the api-ref stuff and I did not start to talk about the functionality of several buttons etc. We should discuss this is a meeting with more people than us.
But the current result does not change my opinion that I am satisfied with the switch to BS5 or we have to spent additional weeks of tuning and drop the work done in the past couple of weeks.

**Comment by gtema** [comment]
oh yeah, api-ref I missed completely

**Comment by tischrei** [comment]
I am not completely against turning to BS5, but it is sad that we waste lot of time in tuning the old crap. That's a bit sad.

**Comment by gtema** [comment]
that is not an argument to stick to ancient toolset and continue fighting with those overrides. Atm there is no need to have "!important" overrides at all (maybe just one). From my pov it is worth of doing switch - this will make like easier in future. Will work on fixing api-ref later today

**Comment by gtema** [comment]
actually all of the changes for api-ref need to be done in otc-api-ref now. I will take care setting up jobs that test this 2 projects together

File: otcdocstheme/theme/otcdocs/static/css/bootstrap.css [hunk 1]
```diff
@@ -1,6757 +1,11266 @@
+@charset "UTF-8";
 /*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * Bootstrap v5.1.3 (https://getbootstrap.com/)
```
Comment (SebastianGode) [inline otcdocstheme/theme/otcdocs/static/css/bootstrap.css]: We do not need the non-minified version

**Review by SebastianGode — CHANGES_REQUESTED** [review]

**Comment by gtema** [comment]
I know, I am still working on it (explicitly this is being deleted), but not pushed due to work on api-ref part

File: doc/requirements.txt [hunk 2]
```diff
@@ -2,10 +2,10 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 # Notes:
-# reno needs openstackdocstheme which needs reno (cycle dep).
-# os-api-ref needs openstackdocstheme which needs os-api-ref (cycle dep).
+# reno needs otcdocstheme which needs reno (cycle dep).
+# otc-api-ref needs otcdocstheme which needs os-api-ref (cycle dep).
```
Comment (outcatcher) [inline doc/requirements.txt:6]: There seems to be no cycle dependency in the second case.

**Review by tischrei — APPROVED** [review]

**Review by tischrei — APPROVED** [review]

**Review by SebastianGode — APPROVED** [review]

**Pull request merged**